### ISSUE TITLE
Return table names in results

### ISSUE THREAD

**Issue by swlkr** [joy-framework/db#2]
The idea is to turn this:

```clojure
(def account (db/find :account 1))

(db/update :account account {:name "new name"})
```

into this:

```clojure
(def account (db/find :account 1))

(db/update account {:name "new name"})

; which is really

(db/update {:id 1 :db/table "account"} {:name "new name"})
```

where the result coming out of a `db/query` has two keys: `:id` and `:db/table`

Insert could work similarly:
```clojure
(db/insert {:db/table "account" :name "new name"})
```

**Comment by swlkr** [comment]
Closed by 97b06a705f14b36190dd26cbbb87b959259dd684

**Comment by swlkr** [comment]
closed
